funcx_endpoint/tests/test_rabbit_mq/conftest.py

```diff
@@ -0,0 +1,22 @@
+import pika
+import pytest
+
+from funcx_endpoint.endpoint.rabbit_mq import ResultQueuePublisher
+
+CONN_PARAMS = pika.URLParameters("amqp://guest:guest@localhost:5672/")
+
+
+@pytest.fixture(scope="module")
```

Comment:
Not sure why the scope is being set to `module` here. I wouldn't generally use that scope at all in a conftest, as I'm not sure what it means, but it might be the same as `session`.

I think you probably want this as a session-scoped autouse fixture. I don't see any usage sites for this, so it's never running as far as I can see.

Reply:
I've updated to session, this was messing up the tests.